### ISSUE TITLE
Typsense import errors fail silently

### ISSUE THREAD

**Issue by stri8ed** [typesense/laravel-scout-typesense-driver#13]
## Description

Attempts to import a model, with an incorrect schema, does not return any errors to the user, despite the underlying JSON API call returning an error along with `succcess` field set to false. This occurs during batch import via `php artisan scout:import` and single model updates. No errors are returned to the user, and no indication is made of the import failing.

## Steps to reproduce

Define a schema on the model like so:

```
public function getCollectionSchema(): array
    {
        return [
            'name' => $this->searchableAs(),
            'fields' => [
                [
                    'name' => 'id',
                    'type' => 'string',
                ]
            ],
            'default_sorting_field' => 'id',
        ];
    }

public function toSearchableArray()
{
  return [
        'id' => $this->id,
  ];
}
```

Bulk import the model via `php artisan scout:import "App\SomeModel\Job"`

## Expected Behavior

The command should fail with an error indicating the failed result, along with the error reason.

## Actual Behavior
Console reports success:
```
Imported [App\SomeModel\Job] models up to ID: 552
```
Yet, no models are actually imported in Typesense. Logging the underlying JSON response within this library, shows the following JSON being returned:

`{"error":"Document's 'id' field should be a string.","success":false}`

## Metadata

**Typsense Version**: 0.21.0

**OS**: Ubuntu 20.04.3

**Comment by jasonbosco** [comment]
This is because the underlying typesense-php library does not throw an exception on imports - this was the earlier design. 

The solve would be to do what the JS library now does, in the typesense-php library: https://github.com/typesense/typesense-js/blob/95bf000f0191b4d3c48854a238bbf55ed5e0483b/src/Typesense/Documents.ts#L203-L213

**Comment by hi019** [comment]
@stri8ed this should be fixed in v5.0.1-rc3. That release includes the latest devloops changes as well. Please try it out!
